### PR TITLE
make openshift-tests wait on a clean image import for images that are used

### PR DESCRIPTION
**Pull request by deads2k**
Some failures can be found here https://search.ci.openshift.org/?search=reading+manifest+latest+in+image-registry.openshift-image-registry.svc%3A5000%2Fopenshift%2Ftools&maxAge=168h&context=1&type=build-log&name=4.10&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

We found this looking at a particular job https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.10-e2e-aws-techpreview-serial/1442861554759372800, but we didn't try chasing down every possible failure.

### CODE REVIEW

**Comment by deads2k** [comment]
/hold

samples operator doesn't watch the tools image.  We'll have to look at something else.  I am curious whether we have permissions here for this change though, so I want to leave this.

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh [hunk 1]
```diff
@@ -260,6 +260,29 @@ echo "$(date) - waiting for clusteroperators to finish progressing..."
...
+
+  count=$((count+1))
+  if (( count > 10 )); then
+    echo "Failed while waiting on imagestream import"
+    exit 1
+  fi
+
+  sleep 60
+done
+echo "$(date) - all imagestreams are imported."
```
Comment (dgoodwin) [inline ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:284]: From: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/22359/rehearse-22359-pull-ci-openshift-installer-release-4.9-e2e-aws-upgrade/1444036535581151232/artifacts/e2e-aws-upgrade/openshift-e2e-test/build-log.txt

Fri Oct  1 21:24:47 UTC 2021 - waiting for non-samples imagesteams to import...
Trying to break loop
Fri Oct  1 21:24:48 UTC 2021 - all imagestreams are imported.

This shows just one second, presumably this is normally all done by this point we're just preventing the flakes when it isn't?

**Review by dgoodwin — COMMENTED** [review]
Seems good to me just a couple small requests/questions.

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh (hunk 1 above)
Comment (deads2k) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:284]: > This shows just one second, presumably this is normally all done by this point we're just preventing the flakes when it isn't?

correct

**Pull request merged**